docs/modals/aside-modal.mdx

```diff
@@ -104,8 +122,39 @@ With the prop <TagDocs text='disableEscapeKeyDown' /> you can disable the closin
 <WindowEditor
   codeString={`import AsideModal from 'dd360-ds/AsideModal'
 
-<AsideModal disableEscapeKeyDown title="Component Aside Modal" open={true} onClose={() => function('function to close the aside modal')} />
-    `}/>
+<AsideModal 
+    disableEscapeKeyDown 
+    title="Component Aside Modal" 
+    open={true} 
```

Comment:
it would be nice to leave only the open, without adding the true, how do you see it @Misael840 ?